### DISCUSSION TITLE
Tracking Portuguese Localization Progress

### DISCUSSION THREAD

**Discussion by edsoncelio** [cncf/glossary#1182] [General]
Issue to track Portuguese (Brazilian) localization progress.
 
## Checklist to translate a term
1. Search if the issue related to the term: https://github.com/cncf/glossary/issues?q=is%3Aissue+is%3Aopen+label%3Alang%2Fpt exists and If no one it's working on it
2. Comment on the issue so one of the approvers will assign it to you, focus on one term per PR.
3. Fork this repo, create your branch from `dev-pt`, make your translation and open a PR with the following name convention: `[Add/Update/Fix] [term path]`, for example:
    `Add content/pt-br/abstraction.md`
4. Open the PR using`dev-pt` as the target branch

For more information about how to contribute, check [this](https://glossary.cncf.io/contribute/).
For more details and any information that you need, please don't hesitate and ask on the [Portuguese slack channel](https://cloud-native.slack.com/archives/C02MWQTJUPL).

**NEW**
For a quick guide about how to start to contribute in Portuguese, check **[this](https://gist.github.com/edsoncelio/d7fdec7f24d5aba87e3e517c672e9eb5)!**
## Localization approvers/reviewers

Any questions or help, just talk with us (here or by slack):

* @brunoguidone 
* @jessicalins
* @edsoncelio 

## Priorities
**UPDATE**: The first version is DONE!
We need at least 10 terms in Portuguese before go to live (v1), below are some suggested terms to work: 

 - [x] (config)`content/pt-br/contribute/_index.md` -> @edsoncelio
 - [x] (config) `content/pt-br/style-guide/_index.md`-> @edsoncelio
 - [x] (config) `content/pt-br/_index.md` -> @edsoncelio
 - [x] (config) `content/pt-br/contributor-ladder/_index.md`  -> @edsoncelio 
 - [x]  `content/pt-br/api_gateway.md` -> @willlsantos 
 - [x]  `content/pt-br/application_programming_interface.md` -> @jessicalins
 - [x]  `content/pt-br/bare_metal_machine.md` -> @MrErlison
 - [x]  `content/pt-br/blue_green_deployment.md` -> @jessicalins 
 - [x]  `content/pt-br/canary_deployment.md` -> @MrErlison
 - [x]  `content/pt-br/client_server_architecture.md` -> @willlsantos 
 - [x]  `content/pt-br/cloud_native_apps.md` -> @MrErlison
 - [x]  `content/pt-br/cloud_native_security.md` -> @MrErlison
 - [x]  `content/pt-br/cluster.md` -> @edsoncelio 
 - [x]  `content/pt-br/container-image.md` -> @MrErlison
 - [x]  `content/pt-br/container.md` -> @edsoncelio 
 - [x]  `content/pt-br/containerization.md` -> @edsoncelio 
 - [x]  `content/pt-br/continuous_delivery.md` -> @MrErlison 
 - [x]  `content/pt-br/continuous_integration.md` -> @MrErlison
 - [x]  `content/pt-br/devops.md` -> @marcelomansur
 - [x]  `content/pt-br/abstraction.md` -> @MatheusPaes
 - [x]  `content/pt-br/agile_software_development.md` -> @MatheusPaes
 - [x]  `content/pt-br/auto_scaling.md` -> @MatheusPaes
 
**To the remaining definitions, you will track the progress by issues.**

### In progress
 - [x]  `content/pt-br/cloud_computing.md` -> @anacunha 
 - [x]  `content/pt-br/cloud_native_tech.md` -> @MrErlison
 - [x] `content/pt-br/data_center.md`  -> @MrErlison
 - [x] `content/pt-br/virtual_machine.md` -> @MrErlison
 - [x] `content/pt-br/virtualization.md` -> @MrErlison
 - [x] `content/pt-br/zero_trust_architecture.md` -> @MrErlison
 - [x] `content/pt-br/serverless.md` -> @luizgribeiro
 - [x] `content/pt-br/TLS(Transport Layer Security).md` -> @MrErlison
 - [x] `content/pt-br/versioncontrol.md` -> @MrErlison
 - [x] `content/pt-br/vertical_scaling.md` -> @MrErlison
 - [ ] `content/pt-br/service.md` -> @MrErlison
 - [x] `content/pt-br/site_reliability_engineering.md` -> @MrErlison
 - [x] `content/pt-br/software_as_a_service.md` -> @MrErlison

**Comment by jessicalins** [comment]
Hello :) 

I'd like to translate `content/pt/application_programming_interface.md` and `content/pt/blue_green_deployment.md`. I'll open PRs for them shortly.

I've tried to translate the content in `pt-BR` - I was wondering if we want to differentiate `pt-pt` from `pt-BR` or for the sake of simplicity we consider both as `pt`? What do you think?

**Comment by edsoncelio** [comment]
Hello, thanks @jessicalins. For now, most of the team members are Brazilians, so I think is better to review/translate in `pt-BR`, later if we found someone to help in `pt-pt` we can discuss more that, but for simplicity, we will consider both `pt`, what do you think?

Ps.: I will assign to you the pages that you are working on, thanks again!

**Comment by jessicalins** [comment]
@edsoncelio sounds good!

**Comment by willlsantos** [comment]
Hi folks, are you ok?

I'd like to translante content/pt/api_gateway.md and content/pt/auto_scaling.md.

**Comment by edsoncelio** [comment]
Thanks @willlsantos, I'll assign it to you if you need some help, just ping us on slack :)

**Comment by matheuspaesp** [comment]
Hello,

I'll tranlsate content/pt/abstraction.md and content/pt/agile_software_development.md
The PR's will be opened soon ;)

**Comment by edsoncelio** [comment]
Thanks @MatheusPaes,I'll assin to you,  if you need some help, just ping us on the slack channel :)

**Comment by willlsantos** [comment]
Hello, 
I'd like to translate content/pt/client_server_architecture.md.

**Comment by Adi9235** [comment]
Hello, @edsoncelio I am new to contributing to cncf projects in this issue we have to translate one term to Portuguese if I am right can I work on content/pt/auto_scaling.md.
Thank You

**Comment by marcelomansur** [comment]
Hi @edsoncelio , I'd like to work on `content/pt-br/devops.md`.

**Comment by MrErlison** [comment]
Hi @edsoncelio check content/pt-br/bare_metal_machine.md for me.

**Comment by MrErlison** [comment]
Hi, @edsoncelio add content/pt-br/container-image.md for me.

**Comment by MrErlison** [comment]
Hi, @edsoncelio add content/pt-br/continuous_delivery.md for me.

**Comment by anacunha** [comment]
@edsoncelio can you please assign `content/pt-br/cloud_computing.md` to me?

Thanks!

**Comment by MrErlison** [comment]
Hi, @edsoncelio add content/pt-br/continuous_integration.md

**Comment by MrErlison** [comment]
Hi @edsoncelio, add content/pt-br/canary_deployment.md for me

**Comment by MrErlison** [comment]
Hello @edsoncelio add content/pt-br/cloud_native_apps.md for me

**Comment by MrErlison** [comment]
Hi @edsoncelio add content/pt-br/cloud_native_security.md

**Comment by MrErlison** [comment]
Hi @edsoncelio add content/pt-br/cloud_native_tech.md #683

**Comment by fharris** [comment]
Hello folks. I would LOVE to join and HELP YOU with the Portuguese definitions!

**Comment by MrErlison** [comment]
Hi @edsoncelio add content/pt-br/data_center.md #682

**Comment by MrErlison** [comment]
Hello @edsoncelio add 
- content/pt-br/virtual_machine.md #716
- content/pt-br/virtualization.md #715
- content/pt-br/zero_trust_architecture.md #714

**Comment by MrErlison** [comment]
Hi @edsoncelio add 
- content/pt-br/TLS(Transport Layer Security).md #759
- content/pt-br/versioncontrol.md #757
- content/pt-br/vertical_scaling.md #758

**Comment by luizgribeiro** [comment]
Hi @edsoncelio could you assign `content/pt-br/serverless.md` to me please?

Cheers

**Comment by MrErlison** [comment]
Hi @edsoncelio for this weekend add these files to me
- content/pt-br/stateless_apps.md #782
- content/pt-br/tightly_coupled_architectures.md #783
- content/pt-br/stateful_apps.md #781

**Comment by MrErlison** [comment]
Hello @edsoncelio add 
- content/pt-br/service.md #810
- content/pt-br/site_reliability_engineering.md #811
- content/pt-br/software_as_a_service.md #812

**Comment by DonatoHorn** [comment]
Hi @edsoncelio, could you please, assign issue 1101 to me.

**Comment by DonatoHorn** [comment]
Hi @edsoncelio, could you please assign to me 
content/pt-br/idempotence.md
content/pt-br/immutable_infraestructure.md
